azure-samples/src/main/java/com/microsoft/azure/management/appservice/samples/ManageWebAppSourceControlAsync.java

```diff
@@ -0,0 +1,268 @@
+        final String planName       = SdkContext.randomResourceName("jplan_", 15);
+        final String rgName         = SdkContext.randomResourceName("rg1NEMV_", 24);
+
+        try {
+
+
+            //============================================================
+            // Create a web app with a new app service plan
+
+            System.out.println("Creating web app " + app1Name + " in resource group " + rgName + "...");
+
+            Observable<?> app1Observable = azure.webApps().define(app1Name)
+                    .withRegion(Region.US_WEST)
+                    .withNewResourceGroup(rgName)
+                    .withNewWindowsPlan(PricingTier.STANDARD_S1)
+                    .withJavaVersion(JavaVersion.JAVA_8_NEWEST)
+                    .withWebContainer(WebContainer.TOMCAT_8_0_NEWEST)
+                    .createAsync()
+                    .flatMap(new Func1<Indexable, Observable<?>>() {
+                        @Override
+                        public Observable<?> call(Indexable indexable) {
+                            if (indexable instanceof WebApp) {
+                                WebApp app = (WebApp) indexable;
+                                System.out.println("Created web app " + app.name());
+                                return Observable.merge(
+                                        Observable.just(indexable),
+                                        app.getPublishingProfileAsync()
+                                        .map(new Func1<PublishingProfile, PublishingProfile>() {
+                                            @Override
+                                            public PublishingProfile call(PublishingProfile publishingProfile) {
+                                                System.out.println("Deploying helloworld.war to " + app1Name + " through FTP...");
+                                                Utils.uploadFileToFtp(publishingProfile,
+                                                        "helloworld.war",
+                                                        ManageWebAppSourceControlAsync.class.getResourceAsStream("/helloworld.war"));
+
+                                                System.out.println("Deployment helloworld.war to web app " + app1Name + " completed");
+                                                return publishingProfile;
+                                            }
+                                        }));
+                            }
+                            return Observable.just(indexable);
+                        }
+                    });
+
+            System.out.println("Creating another web app " + app2Name + " in resource group " + rgName + "...");
+            System.out.println("Creating another web app " + app3Name + "...");
+            System.out.println("Creating another web app " + app4Name + "...");
+
+            Observable<?> app234Observable = azure.appServices().appServicePlans()
```

Comment:
Also printing that website is created - can be moved to map part of individual observables.